testsuite/testplanet/copy_test.c

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2022 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+#include <stdlib.h>
+#include <string.h>
+
+#include "../require.h"
+#include "helpers.h"
+#include "uplink.h"
+
+void handle_project(UplinkProject *project);
+
+int main()
+{
+    with_test_project(&handle_project);
+    return 0;
+}
+
+void handle_project(UplinkProject *project)
+{
+    {
+        UplinkBucketResult bucket_result = uplink_ensure_bucket(project, "alpha");
+        require_noerror(bucket_result.error);
+        uplink_free_bucket_result(bucket_result);
+    }
+
+    size_t data_len = 5 * 1024; // 5KiB;
+    uint8_t *data = malloc(data_len);
```

Comment:
malloc without free.

(maybe you can allocate the data on the stack if the compiler allows it)